core/_neat.scss

```diff
@@ -0,0 +1,11 @@
+// Neat 2.0.0.alpha.0
+// http://neat.bourbon.io
+// Copyright 2015 thoughtbot, inc.
+// MIT License
+
+@charset "UTF-8";
+
+@import "neat/settings/settings";
+
+@import "neat/mixins/grid-column";
```

Comment:
Something I’d like to do with Bourbon—and would also like to see here—is moving away from a directory structure named after features of Sass, i.e. `mixins`. Can we generalize this to a `library` (open to other ideas!) directory, which are just the helpers and tools of Neat, and this way it can contain functions and all other types of Sass features and the name will still make sense.